src/routes/routes.js

```diff
@@ -1,54 +1,59 @@
-import { signin, signup } from '../controllers/authController';
+const express = require('express');
+
 import {
-  validateSigninFormData,
-  validateSignupFormData,
-  validUser,
-  validateCommentData,
-  validateInvite,
-  validateInviteId,
-  validateUUID,
-  verifyUniqueUser
-  authenticateUserToken,
-  validateAdmin,
-  validateUserById,
-  validateUserId
+    validateSigninFormData,
+    validUser,
+    validateCommentData,
+    validateInvite,
+    validateInviteId,
+    authenticateUserToken,
+    validateAdmin,
+    validateUserById,
+    validateUserId
 } from '../middlewares/middlewares';
-
-import { getComments, createComment } from '../controllers/commentController';
 import {
-  authenticateUserToken,
-  verifyUniqueUser
-} from '../middlewares/authentication';
+    signin
+} from '../controllers/authController';
+import {
+    getComments,
+    createComment
+} from '../controllers/commentController';
 import { upvoteInvite } from '../controllers/upvoteController';
 import { deleteInvite } from '../controllers/inviteController';
 import { blockUser, getUsers } from '../controllers/userController';
 
+
+
+const jobInviteRouter = express.Router();
+jobInviteRouter.route('/')
+    .get((req, res) => {
+        res.render('jobInvites')
+    })
+module.exports = jobInviteRouter
+
+
+
```

Comment:
Absolutely unacceptable... Read the comments in this file before you code.

it should be `app.get('/post', (req, res) => res.render('jobInvites'));` and this line should be placed after the line with `app.get('/post', (req, res) => res.render('userPost'));`